package.json

```diff
@@ -48,6 +48,8 @@
   },
   "dependencies": {
     "base-64": "^0.1.0",
+    "jsrsasign": "^8.0.12",
```

Comment:
I'd like to take a quick look at this library before we use it. Can we pin the version?

Reply:
Library looks ok, but let's pin it at 8.0.12 since it hasn't been actively maintained since 2 years ago.

Reply:
Fixed version to 8.0.12